### PR TITLE
documentation for using proxy

### PR DESCRIPTION
**Pull request by rockb1017**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,10 @@ autodetect:
   prometheus: true
 ```
 
+## Proxy Support
+
+Otel collector in this chart support [proxy environment variables](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter#proxy-support). You can set these proxy environment variables by setting `.Values.otelAgent.extraEnvs` for agent daemonset or `.Values.otelCollector.extraEnvs` for collector deployment.
```
Comment (dmitryax) [inline README.md:294]: We have a config example for using proxy https://github.com/signalfx/splunk-otel-collector-chart/tree/main/examples#route-telemetry-data-through-a-proxy-server . Please add a link to it

Comment (dmitryax) [reply README.md:294]: Also I think this should go in `## Advanced configuration` section

Comment (rockb1017) [reply README.md:294]: oh i didn't see that example part. i think that is good enough. We shouldn't need another doc at README. what do you think

Comment (dmitryax) [reply README.md:294]: Sounds good to me. And we already have links to examples in the README so we probably can close this PR

**Pull request closed without merging**